qe_metrics/libs/database.py

```diff
@@ -127,3 +134,44 @@ class JiraIssues(DB_CONNECTION.Entity):  # type: ignore
         customer_escaped = orm.Required(bool)
         date_created = orm.Required(date)
         last_updated = orm.Required(date)
+        date_record_modified = orm.Required(datetime, default=datetime.now)
```

Comment:
why do we need both this and `last_updated`?

Reply:
@rnetser `last_updated` maps to the "Updated" date in a Jira issue. The `date_record_modified` is the when the record is placed in the database or updated by our script. There is the potential to retrieve an issue that was "Updated" two weeks ago but was added to the database today. I was planning on using `date_record_modified` in the data retention clean up process. Do you think I should just use `last_updated`?